File: java/moderate/(13) remove-characters.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            
            String[] splitLine = line.split(", ");
            String sentance = splitLine[0];
            String charsToRemove = splitLine[1];

            for (char ch: charsToRemove.toCharArray()){
                sentance = sentance.replaceAll(String.valueOf(ch), "");
            }

            System.out.println(sentance);
        }
    }
}

File: python/moderate/(13) remove-characters.py
import sys

with open(sys.argv[1], 'r') as f:
    for line in f.readlines():
        line_list = line.split(', ')
        sentance = line_list[0]
        chars_to_remove = line_list[1]

        for char in chars_to_remove:
            sentance = sentance.replace(char, '')

        print sentance
